Monika After Story/game/script-story-events.rpy

```diff
@@ -2489,3 +2489,49 @@ label mas_notification_windowreact:
             show monika 5ttu at t11 zorder MAS_MONIKA_Z with dissolve
             m 5ttu "...right?"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_change_to_def",
+            unlocked=False
+        )
+    )
+
+label mas_change_to_def:
+    # sanity check for an extremely rare case where player dropped below happy
+    # closed game before this was pushed and then deleted json before next load
+    if store.monika_chr.clothes == store.mas_clothes_def:
+        return
+
+    m 1eka "Hey [player], I miss my old school uniform..."
+    m 3eka "I'm just going to go change, be right back..."
+    
+    $ outfit = "def"
+    call mas_clothes_change
+
+    m "Okay, what else should we do today?"
+
+    # remove from event list in case PP and ch30 both push
+    $ mas_rmallEVL("mas_change_to_def")
+    return "no_unlock"
+
+label mas_clothes_change:
```

Comment:
```suggestion
label mas_clothes_change(outfit):
```
change label to be functionized, then call this like `call mas_clothes_change("def")`